src/matlab2tikz.m

```diff
@@ -3289,9 +3295,10 @@ case size(cData,2) % specific y-data is given
         [xDataPlot, yDataPlot] = deal(xData, yData);
     end
 
-    drawOpts = opts_print(m2t, drawOptions, ',');
-    [m2t, table ] = makeTable(m2t, '', xDataPlot, '', yDataPlot);
-    str = sprintf('\\addplot[%s] plot table[row sep=crcr] {%s};\n', drawOpts, table);
+    [m2t, table, tabOpts] = makeTable(m2t, '', xDataPlot, '', yDataPlot);
+    str = sprintf('\\addplot[%s] plot table[%s] {%s};\n', ...
+                 opts_print(m2t, drawOptions, ','), ...
+                 opts_print(m2t, tabOpts, ','), table);
```

Comment:
Beatify by unifying `tabOpts` and `drawOptions` to both `<x>Opts` or `<x>Options`. This eases search (and replace) for debugging. Applies to the whole code preferably and not only this sub-function.